modules/ipi-preparing-reinstall-cluster-bare-metal.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+// //installing/installing_bare_metal_ipi/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
+
+[id="ipi-preparing-reinstall-cluster-bare-metal_{context}"]
+
+= Preparing to reinstall a cluster on bare-metal
+Before you reinstall a cluster on bare-metal, you must perform cleanup operations.
+
+.Procedure
+. Remove or reformat the disks for the bootstrap, master, and worker nodes. If you are working in a hypervisor environment, you will need to re-add any disks you removed. You are now starting with clean disks for the bootstrap, master, and worker nodes.
+. Delete the artifacts that the previous installation generated:
+[source,terminal]
+----
+$ cd ; /bin/rm -rf auth/ bootstrap.ign master.ign worker.ign metadata.json .openshift_install.log .openshift_install_state.json
+----
```

Comment:
```suggestion
. Delete the artifacts that the previous installation generated:
+
[source,terminal]
----
$ cd ; /bin/rm -rf auth/ bootstrap.ign master.ign worker.ign metadata.json \
.openshift_install.log .openshift_install_state.json
----
```
Refer to the sections on [lists](https://github.com/openshift/openshift-docs/blob/f6e9d0618b5498bd459e47ed8932a76b747a83cb/contributing_to_docs/doc_guidelines.adoc#lists) and [breaking up long lines](https://github.com/openshift/openshift-docs/blob/f6e9d0618b5498bd459e47ed8932a76b747a83cb/contributing_to_docs/doc_guidelines.adoc#lists) in the OSDOCS guidelines.

Reply:
Done. Will revise standards.